tests/resources/hadoop-hive-kerberos.json

```diff
@@ -0,0 +1,31 @@
+{
+  "instances": 1,
+  "cpus": 1,
+  "mem": 4096,
+  "user": "root",
+  "container": {
+    "type": "MESOS",
+    "docker": {
+      "image": "susanxhuynh/cdh5-hive-kerberos:latest",
```

Comment:
Let's add this to the list of docker images that we need to move to the `mesosphere` account.